app/assets/stylesheets/application.scss

```
@@ -6,11 +6,23 @@
 
 /* 全体 */
 
+body {
+  padding-top: 70px;
```

ナビバーの縦幅にあわせておきましょうか。

```suggestion
  padding-top: 56px;
```